ruby/rails/security/brakeman/model-serialize.yaml

```diff
@@ -0,0 +1,30 @@
+rules:
+- id: model-serialize
```

Comment:
this is effectively a duplicate of an SCA rule - @adamberman how do we want to handle this sort of coverage in Brakeman ports?

Reply:
nm, this is the *first* version of messing up DB serialization.  Everything old is new again, as usual.

Reply:
ah I see, gotcha